src/model_pool.hpp

```diff
@@ -280,6 +280,19 @@ namespace wr
 
 		ModelData* data = loader->Load(path);
 
+		if (flip_normals)
+		{
+			for (auto* meshes : data->m_meshes)
+			{
+				for (auto& normals : meshes->m_normals)
+				{
+					normals.x = -1.f * normals.x;
```

Comment:
I think you can use the tangent to calculate whether the normal should be flipped.